packages/blitz-rpc/src/loader-server.ts

```diff
@@ -90,17 +100,20 @@ export async function recursiveFindResolvers(
       const pathStat = await promises.stat(absolutePath)
 
       if (pathStat.isDirectory()) {
```

Comment:
here is where you should check for node_modules and abort. because we don't want to call `recursiveFindResolvers` on node_modules